src/utils/utils.tsx

```diff
@@ -0,0 +1,39 @@
+import { Todo } from '../types/Todo';
+
+export enum FilterEnum {
+  ALL = 'all',
+  COMPLETE = 'completed',
+  ACTIVE = 'active',
+
+}
```

Comment:
you can split th logic

create a folder `enums` and put this `FilterEnum` there